sbol_utilities/helper_functions.py

```diff
@@ -357,3 +357,12 @@ def collector(obj: sbol3.Identified):
     with cached_references(doc):
         doc.traverse(collector)
     return outgoing
+
+def find_feature(doc: sbol3.Component,f:sbol3.Feature) -> sbol3.Feature:
```

Comment:
This isn't actually addressing #185, as one has to already have the feature in order to find it.
In the issue, the example is looking for a feature whose identity is known, but not that actual object.

Reply:
yeah, got it

what if I used this line of code? 
` if i.instance_of.split('/')[-1] == f.identity.split('/')[-1]:`
using `.identity` with the feature we input and find the feature name then comparing it with the same feature if it exist using `.instance_of`

Reply:
That doesn't address the question of the function's interface.